libappstream-glib/as-app.c

```diff
@@ -6323,7 +6325,7 @@ as_app_parse_appdata_file (AsApp *app,
 			     filename, error_local->message);
 		return FALSE;
 	}
-	data = g_bytes_new_take (g_steal_pointer (&data_raw), len);
+	data = g_bytes_new_take (g_steal_pointer (&data_raw), len + 1);
```

Comment:
This is wrong, this would read beyond the allocation.

Reply:
> and a nul is added to the end

I've learned something today... okay, so that hunk is good, but could you add a comment to help 5-year-old-me for the future?